LoopKitUI/Views/Settings Editors/TherapySettingsView.swift

```diff
@@ -130,7 +130,7 @@ extension TherapySettingsView {
     private var correctionRangeSection: some View {
         SectionWithEdit(isEditing: $isEditing,
                         title: LocalizedString("Correction Range", comment: "Correction Range section title"),
-                        footer: EmptyView(),
+                        descriptiveText: "Correction range is the glucose value (or range of values) that you want Tidepool Loop to aim for in adjusting your basal insulin.",
```

Comment:
(I think there was just a discussion about unifying this text with the setting description; I'll assume that's for a future refactoring)

Reply:
Yep!  I haven't gotten an answer on that yet, so I'm forging ahead until then.